tasks/task534_farstail_entailment.json

```diff
@@ -8,31 +8,32 @@
     "Categories": [
         "Classification"
     ],
-    "Definition": "In this task, you're given a pair of sentences in Persian Language written in persian alphabet.Your job is to choose whether the two sentences clearly agree (entailment)/disagree (contradiction) with each other, or if this cannot be determined (neutral). Your answer must be in the form of the letters E, C, and N respectively. The sentences have been separated by a new line character.",
     "Input_language": "fa",
     "Output_language": "en",
+    "Instruction_language" : "en",
+    "Definition": "In this task, you're given a pair of sentences in Persian Language written in persian alphabet. Your job is to choose whether the two sentences agree (entailment), disagree (contradiction) or neither (neutral). Your answer must be in the form of the letters E, C, and N respectively. The sentences have been separated by a new line character.",
```

Comment:
I think it's better to drop `in Persian Language written in persian alphabet.`, because we already know that input language is Persian.
Please add a comma in `(contradiction) or neither` after (contradiction) and in ` N respectively` after "N".
It will be more clear if you explain which word each letter is referring to.

Reply:
I think we should specify the alphabet we are using because some translation tasks use the native alphabet and the English alphabet ([example](https://github.com/swarooprm/CSE576HW/blob/69ed0eabf66355f63d97b8d93301d7b5aa72c582/tasks/Task361_tamil_offenseval_dravidian_classification.json)). but I will remove it if you think I should

Reply:
This makes sense. So please replace it with `in the Persian Language written in the Persian alphabet`.
Yes, As far as I know, we need a comma before respectively, when it comes at the end of the sentence.